run.py

```diff
@@ -300,13 +306,24 @@ def reindex(since=None, batch_size=None, force_indexing=False, update_solr=True,
         if args.filename:
             print 'deleting bibcodes from file via queue'
             bibs = []
-            with open(args.filename) as f:
+            with open(args.filename, 'r') as f:
                 for line in f:
                     bibcode = line.strip()
                     if bibcode:
                         tasks.task_delete_documents(bibcode)
         else:
             print 'please provide a file of bibcodes to delete via -n'
+
+    elif args.augment:
+        if args.filename:
+            with open(args.filename, 'r') as f:
+                for line in f:
+                    bibcode = line.strip()
+                    if bibcode:
+                        # read db record for current aff value, send to queue
+                        # aff values omes from bib pipeline
+                        
+                        app.request_aff_augment(bibcode)
```

Comment:
so this will establish connection for every bibcode that ever arrives and is processed by the pipeline; that won't work - you should rather use the forward mechanism like the other queues; they establish connections once (when app is created) and then use it to forward messages to master

or establish connection and keep it in the app instance

how will the task be triggered? would it make sense to ask for an aff update when a bibcode protobuf is received?

Reply:
the current code in run.py is only used during testing, we do not yet have a cron job to bulk update changed affiliations.  run.py will be fixed next week in another PR that also addresses the "wrong destination" issue.